backend/db/config.py

```diff
@@ -99,17 +105,21 @@ def get_pg_connect_url(local=False):
 DERIVED_TABLE_DEPENDENCY_MAP = {
     # Dependencies figured out
     # - tables
-    'all_csets': ['code_sets', 'omopconceptset', 'concept_set_container', 'omopconceptsetcontainer', 'concept_set_counts_clamped', 'codeset_counts'],  # omopconceptset is all lowercase in DB but referred to as OMOPConceptSet in DDL and enclave object API
+    # all_csets: omopconceptset is all lowercase in DB but referred to as OMOPConceptSet in DDL and enclave object API
+    'all_csets': ['code_sets', 'omopconceptset', 'concept_set_container', 'omopconceptsetcontainer',
+                  'concept_set_counts_clamped', 'cset_term_usage_rec_counts',  'codeset_counts', 'researcher'],
     'codeset_counts': ['members_items_summary'],
     'codeset_ids_by_concept_id': ['cset_members_items'],
     'concept_ancestor_plus': ['concept_ancestor', 'concepts_with_counts'],
+    'concept_graph': ['concept_ancestor'],
     'concept_ids_by_codeset_id': ['cset_members_items'],
     'concept_relationship_plus': ['concept_relationship', 'concepts_with_counts'],
     'concepts_with_counts': ['concepts_with_counts_ungrouped'],
     'concepts_with_counts_ungrouped': ['concept', 'deidentified_term_usage_by_domain_clamped'],
     'cset_members_items': ['concept_set_members', 'concept_set_version_item'],
+    'cset_term_usage_rec_counts': ['concept_set_members', 'concepts_with_counts'],
```

Comment:
## Add `cset_term_usage_rec_counts ` to `DERIVED_TABLE_DEPENDENCY_MAP`
@Sigfried Actually, I see that in the DDL this is supposed to be dropped after creating `all_csets`. Is it a mistake that this is here or are you using it? If so, are you just using it temporarily?
If using it, I need to edit the DDL for `all_csets` to remove the drop, and make a new DDL for this table.
If not using, we should delete the table and remove from this config.

Reply:
It's important to know the answer to this question, as I've kept the table but since had to remove from the config because this was breaking the counts/vocab refresh (possibly also the main refresh but probably not).

Reply:
We went with (a), dropping the table. It just needed to be dropped from the DB 1x. In the future, the DDL automatically drops it upon creation.